### PR TITLE
0.61.3 - Add structured content methods for Carbon Accordion

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: app/views/schools/analysis/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,29 @@
 <% content_for :page_title, @title %>
-<%= link_to 'Back', school_analysis_index_path(@school), class: 'btn btn-rounded mt-3' %>
-<% @content.each do |content| %>
-  <%= render content[:type].to_s, content: content[:content], school: @school %>
+<%= link_to 'Back', school_analysis_index_path(@school), class: 'btn btn-rounded mt-3 mb-2' %>
+
+<% if @structured_content %>
+  <div class="accordion" id="accordionExample">
```
Comment (jamesalmond) [inline app/views/schools/analysis/show.html.erb]: Better id?

File: app/services/alerts/adapters/analytics_adapter.rb [hunk 2]
```diff
@@ -20,6 +20,16 @@ def benchmark_dates
         analysis_object.benchmark_dates(@analysis_date)
       end
 
+      def has_structured_content?
```
Comment (jamesalmond) [inline app/services/alerts/adapters/analytics_adapter.rb]: Should we add these methods to the system adapter too?

File: app/views/schools/analysis/show.html.erb (hunk 1 above)
Comment (jamesjefferies) [reply app/views/schools/analysis/show.html.erb]: Yes, sure

File: app/services/alerts/adapters/analytics_adapter.rb (hunk 2 above)
Comment (jamesjefferies) [reply app/services/alerts/adapters/analytics_adapter.rb]: Yes, I think we should

**Pull request closed without merging**